src/java_bytecode/java_bytecode_convert_method.cpp

```diff
@@ -1123,28 +1093,23 @@ codet java_bytecode_convert_methodt::convert_instructions(
 
   while(!working_set.empty())
   {
-    std::set<unsigned>::iterator cur=working_set.begin();
-    address_mapt::iterator a_it=address_map.find(*cur);
-    CHECK_RETURN(a_it != address_map.end());
+    auto cur = working_set.begin();
+    auto a = address_map.at(*cur);
```

Comment:
I don't think this is semantics preserving: the previous code would result in modifications to the mapped element, while all those now happen on a copy. `auto` is dangerous.